virt/virtual_machines/cloning_vms/virt-cloning-vm-disk-into-new-datavolume-block.adoc

```diff
@@ -16,6 +16,10 @@ Cloning operations between different volume modes are supported, such as cloning
 However, you can only clone between different volume modes if they are of the `contentType: kubevirt`.
 ====
 
+[TIP]
+====
+When preallocation is enabled globally, or for a single data volume, the containerized data importer (CDI) preallocates disk space during cloning. For more information, including benefits to write performance, see xref:../../../virt/virtual_machines/virtual_disks/virt-using-preallocation-for-datavolumes.adoc#virt-using-preallocation-for-datavolumes[Using preallocation for data volumes].
```

Comment:
Suggestion: "The Containerized Data Importer (CDI) preallocates disk space during cloning when preallocation mode is enabled globally for the cluster or for a specific data volume. For more information about preallocation, ...."

In any case s/containerized data importer/Containerized Data Importer

Applies to all files

Reply:
Capped CDI. I reworded the sentence to: 

_When you enable preallocation globally, or for a single data volume, the Containerized Data Importer (CDI) preallocates disk space during cloning. Preallocation enhances write performance. For more information, see....._

Hi @sjhala-ccs - Vale passed this version but not "The Containerized..." version, which it flagged as passive voice. My main problem with that suggestion is it starts the sentence by talking about the CDI and what it does rather than what the user can/should do. So, I'd like to go with the version I submitted.